deploy/src/main/resources/conf/app.conf

```diff
@@ -19,7 +19,7 @@ smart-gcal {
 lucuma-sso {
     service-token = "<SSO_SERVICE_JWT environment variable not defined!>"
     service-token = ${?SSO_SERVICE_JWT}
-    sso-url = "https://sso.gpp.lucuma.xyz" # Staging SSO Server and its public key
+    sso-url = "https://sso-dev.gpp.lucuma.xyz" # Staging SSO Server and its public key
     public-key = "-----BEGIN PGP PUBLIC KEY BLOCK-----\n\nmQINBGQ0QyMBEACytn5gMtKPfRIQc+Wbf4B4tizVq8Kuxe5IUkdULydEWuTCiaeV\nk8Gghs+FGjiFx3mCS9Qa8pj6VaajMcqFIkG/QuDdhhHVBBHG8c8eyzKowe08KlLZ\n3qrKJhVm1O/45a4Pp6VN5o+tbIkiXSS5bVgMJJMEin7JyrL96qjG301MIxBfpI49\nIdgM98AzleE47J/Yl4GjxJ8DZfFKW4SoFrgAWrmeNaz81gTJkMVtAbrnvIGdM207\nwVM3fd0yfQHVnzLvrmcXLDtYFgrAPZOJUseREq4Ab3KS9c/JHcxYvbYGufZsmbSy\nNVDZYeQEPPYdwLuybMKkjFSDpTPAgNjh/oqEEy2yzMPUoigKwMsvVd0etvg/mJjW\nc96pSA/6Bo8/v8uwuqV7FZl8NY9rK3ECoxUbMXtHKa9mZ4EGNB+PpD/a7O3QQ38T\nufvV15ARtizcAmAEglOJmzC/f8PjygyImt3QGfIzdv+JXQp8Ww06lnI7rij3M/gk\nJQBoxHrUszbdhE/E5JvqudNSy4KNFFH2gnyx55QmqHis3Cai6tbXvjpIFGzI1reA\nsxYKC2iEIl3jVv9qLZPVYju6vFAaFL4vPbi9OQf7yHjLZ97eqO4VANg162xYO+sr\n3az0rywQS6C3cl8oTXC3Fx7M2UvSVjQjif/ji7Rcg53prB3Cj60conawiQARAQAB\ntCNMdWN1bWEgU1NPIDxyb2Iubm9ycmlzQG5vaXJsYWIuZWR1PokCVwQTAQgAQRYh\nBHyvVN9WDKRIVlSY35f9xbLdAQ96BQJkNEMjAhsDBQkDwmcABQsJCAcCAiICBhUK\nCQgLAgQWAgMBAh4HAheAAAoJEJf9xbLdAQ96aYwP/0OgEaehsGdwjB+AGYR6k897\ntVsgd6vWlQDAysGwVtUFT8ZpjDWy90qKyd7Syj8N36v70XpeCtXuX1GieftL5Vna\nRdpncWlhTl4oMAH/5/ZLCSsVTE/cLjRCR0V5hjow2tbG2a+XOX85eSTIrNuarVfz\n1jCqY01cGbS7+OJLdgGcoF+xDLISkJxzKIZcTfMIdNXcAPRzNjPQ3v0DiUfUUALD\neKFcKPdAtsEr5GT7OzRh4PrOzS41lfLV0Zo68TA/POWUDJKw/p2eROs0K5utj3BP\nP0/OB1xSns1oyBK1iV5NS327DnvhtmVwJZ3ypKe5x0xfwBoEIv7+GNrS2B1aT5Bt\nPUmWPaMxwrz11ok6DOi8dZLMPzzHelXgOlxZ4aqZdi/+EMWAb0MupDNk4Tqp3LO4\ny6bzY/J+SlrOFueoXKmFL7u9Y/3NMKS/u8PzHUvR1mRowbKBx86oBNNcDpaUqWdE\nfeq0loPI2eiayic+gSWFSdIwEgjfs2yN4WwCR+Wed9PBm18/7kh3zn8CA+vivWJV\nwCUwNi8gwTP2BEfAB801n8v327tW9aQVv74WmnbDLaO3nepupHn+Vt2SLnUwaOGN\nKQeP3PSZxqczHCnyn++TI2sVDa07EZgQg1VshfuE+BVTLa72D73CZ9zK04zbFlBm\nmyXfbbs0K6ZHIj9KIVk3\n=jo/I\n-----END PGP PUBLIC KEY BLOCK-----"
```

Comment:
Doesn't the public key change when we change the server?

Reply:
Good question, I was able to login though